backend/migrations/20230703130438_food.up.sql

```diff
@@ -1,5 +1,5 @@
 -- Add up migration script here
-CREATE TYPE meal_type AS ENUM ('VEGAN','VEGETARIAN','BEEF','BEEF_AW','PORK','PORK_AW','FISH','UNKNOWN');
```

Comment:
I do not think you are even allowed to insert whitespace, as the files hashes are compared.